pytition/pytition/settings/base.py

```diff
@@ -27,7 +27,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['0.0.0.0']
```

Comment:
Could you set 
```python
ALLOWED_HOSTS = ['0.0.0.0', '127.0.0.1', 'localhost', '[::1]']
```
So that it keeps working with localhost and 127.0.0.1 when running outside Docker?
See https://docs.djangoproject.com/fr/2.1/ref/settings/#allowed-hosts for the details why this is necessary to add those 3 extra when the variable isn't empty.

Reply:
Apart from this last change, I'm OK to merge the PR when last comment is addressed :) Thanks!